public/src/components/calls/pre_call_btn_grp.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { ButtonGroup, Button } from 'react-bootstrap';
+
+export default class PreCallButtonGroup extends Component {
+  constructor(props) {
+    super(props);
+    const handlers = ['handleStartCallingClick', 'handleStopCallingClick', 'handleSkipClick', 'handleBadNameClick'];
+    handlers.forEach((handler) => {
+      this[handler] = this[handler].bind(this);
+    });
+  }
+
+  handleStartCallingClick() {
+    const { updateCallStatus } = this.props;
+    updateCallStatus('IN_PROGRESS');
+  }
+
+  handleSkipClick() {
+    const { call_id, user_id, campaign_id, releaseCall, nextCall } = this.props;
+    releaseCall(user_id, campaign_id, call_id);
+    nextCall(user_id, campaign_id);
+  }
+
+  handleBadNameClick() {
+    const { call_id, user_id, campaign_id, updateAttempt } = this.props;
+    const outcome = 'BAD_NUMBER';
+    updateAttempt(user_id, campaign_id, call_id, outcome);
```

Comment:
This should also go ahead and assign us a new call?

Reply:
Yep, that happens in the .then block of updateCallAttempt()